tsl/test/expected/gapfill.out

```diff
@@ -1136,28 +1149,28 @@ SELECT
 FROM (VALUES (1),(2)) v(t)
 WHERE true AND true
 GROUP BY 1;
-ERROR:  invalid time_bucket_gapfill argument: finish cannot be NULL
+ERROR:  invalid time_bucket_gapfill argument: could not infer finish boundary from WHERE clause
 -- NULL finish and no usable time constraints
 SELECT
   time_bucket_gapfill(1,t,1,NULL)
 FROM (VALUES (1),(2)) v(t)
 WHERE true AND true
 GROUP BY 1;
-ERROR:  invalid time_bucket_gapfill argument: finish cannot be NULL
+ERROR:  invalid time_bucket_gapfill argument: could not infer finish boundary from WHERE clause
 -- expression with column reference on right side
 SELECT
   time_bucket_gapfill(1,t)
 FROM (VALUES (1),(2)) v(t)
 WHERE t > t AND t < 2
 GROUP BY 1;
-ERROR:  invalid time_bucket_gapfill argument: start cannot be NULL
+ERROR:  invalid time_bucket_gapfill argument: could not infer start boundary from WHERE clause
 -- expression with multiple column references
 SELECT
   time_bucket_gapfill(1,t1+t2)
 FROM (VALUES (1,2),(2,2)) v(t1,t2)
 WHERE t1 > 1 AND t1 < 2
 GROUP BY 1;
-ERROR:  invalid time_bucket_gapfill argument: start cannot be NULL
+ERROR:  invalid time_bucket_gapfill argument: ts needs to refer to column if no start or finish is supplied
```

Comment:
I don't understand this error message.

Reply:
ts is the 2nd arg to time_bucket_gapfill, when start/finish is not supplied as argument, ts needs to be a column reference so we can match it up with the column constraints in the where clause